src/core.jl

```diff
@@ -933,6 +944,8 @@ function solve(p::NomadProblem, x0::Vector{Float64})
             add_nomad_param!(c_nomad_problem, "VNS_MADS_SEARCH_TRIGGER " * string(p.options.vns_mads_search_trigger))
         end
 
+        add_nomad_param(c_nomad_problem, "NB_THREADS_PARALLEL_EVAL", p.options.bb_max_nb_threads)
```

Comment:
By convention, we try to remain close to the NOMAD parameters (otherwise, we could get lost). So, I would suggest you to use ``nb_threads_parallel_eval``.